contracts/BaseStrategy.sol

```diff
@@ -13,6 +13,7 @@ struct StrategyParams {
     uint256 minDebtPerHarvest;
     uint256 maxDebtPerHarvest;
     uint256 lastReport;
+    uint256 lockedProfit;
```

Comment:
That is the part that took me the longest to debug why harvest was withdrawing all the funds from the strategy, indeed lockedProfit=0 was used instead of totalDebt because I forgot to add this line here. Hopefully, interface will be possible in the future.